superset/views/core.py

```diff
@@ -731,6 +734,7 @@ def explore(  # pylint: disable=too-many-locals,too-many-return-statements
         )
 
     @api
+    @event_logger.log_this
```

Comment:
The order of the decorators here seem random

Reply:
Where do you think it belongs, before `@expose`?